cost/alibaba/Alibaba_Single_File/Alibaba_Monthly.pt

```diff
@@ -0,0 +1,283 @@
+name "Alibaba Cloud Common Bill Ingestion Aggregate"
```

Comment:
We would want to support daily granularity in basically all cases. Therefore a Monthly granularity policy is not needed in the Catalog.